monai/networks/layers/factories.py

```diff
@@ -242,6 +245,25 @@ def sync_batch_factory(_dim) -> Type[nn.SyncBatchNorm]:
     return nn.SyncBatchNorm
 
 
+@Norm.factory_function("instance_nvfuser")
+def instance_nvfuser_factory(dim):
```

Comment:
I feel this function name is not very clear? and please add the GitHub link of `apex` to the doc-string. Because we can't install `apex` by `pip install` directly as an optional package, need to let users know how to install it if not using NVIDIA docker image.

Thanks.

Reply:
Thanks Nic!

Reply:
I am not sure actually..`apex` seems should not be in the function name..
Hi @wyli , do you have any suggestion? Maybe just use the original `instance_nvfuser_factory`?

Thanks.

Reply:
thanks, I think `instance` and `instance_nvfuser` make sense here.

Reply:
I'm pretty sure you've guys talked it through, but my two cents:

I would imagine that `instance_nvfuser_factory` would be merged with `instance_factory`, and the logic would be the following:
`if has_nvfuser and dim == 3: return InstanceNorm3dNVFuser`
`else: return types[dim - 1]`.

This way a user will simply request `InstanceNorm` and it transparently will try to load the NVFuser implementation, and will fallback to native if unsuccessful. What do you think?

Reply:
Hi @mmarcinkiewicz , thanks for the suggestion. So far, `InstanceNorm3dNVFuser` does not support TorchScript because of using a customized autograd function (Please check my comments below for more details). In addition, it also requires to use cuda in a non-Windows env. I'm not sure if combing it into `instance_factory` with these checks is a good way. Hi @Nic-Ma @wyli , what do you think? Thanks!